test/unit/mdc-auto-init/mdc-auto-init.test.js

```diff
@@ -95,3 +96,47 @@ test('#register warns when registered key is being overridden', () => {
 
   td.verify(warn(contains('(mdc-auto-init) Overriding registration')));
 });
+
+test('#emit dispatches a custom event with supplied data', () => {
+  const handler = td.func('eventHandler');
+  let evt = null;
+
+  td.when(handler(td.matchers.isA(Object))).thenDo((evt_) => {
+    evt = evt_;
+  });
+
+  const data = {evtData: true};
+  const type = 'MDCAutoInit:End';
+
+  document.addEventListener(type, handler);
+  emit(type, data);
+
+  assert.isOk(evt !== null);
+  assert.equal(evt.type, type);
+  assert.deepEqual(evt.detail, data);
+});
+
+test('#emit dispatches an event with supplied data - custom events not supported', () => {
```

Comment:
Same thing here. Consider rewording '#mdcAutoInit dispatches a *MDCAutoInit:End* event when initialization of all components is complete - custom events not supported'?